### PR TITLE
Issue #238 fix

### PR DESCRIPTION
**Pull request by MKiselev**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #238

### CODE REVIEW

File: generators/model/Generator.php [hunk 1]
```diff
@@ -242,6 +242,29 @@ public function generate()
     }
 
     /**
+     * Generates the properties for the specified table.
+     * @param \yii\db\TableSchema $table the table schema
+     * @return array the generated properties (property => type)
+     */
+    public function generateProperties($table)
```
Comment (samdark) [inline generators/model/Generator.php]: Should it be public?

File: generators/model/Generator.php [hunk 2]
```diff
@@ -242,6 +242,29 @@ public function generate()
     }
 
     /**
+     * Generates the properties for the specified table.
+     * @param \yii\db\TableSchema $table the table schema
+     * @return array the generated properties (property => type)
```
Comment (samdark) [inline generators/model/Generator.php]: Need `@since`

File: generators/model/Generator.php (hunk 1 above)
Comment (MKiselev) [inline generators/model/Generator.php]: @samdark no, but methods e.g. generateLabels(), generateRules() also are public, but used only in current class.

Comment (samdark) [inline generators/model/Generator.php]: Let's change visibility to private if it's not used anywhere. Or to protected if it could be overridden.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**